### PR TITLE
test: DRY and tidy up card tests

### PR DESCRIPTION
**Pull request by colincclark**
- mostly to do with ordering and naming stories and tests for consistency

### CODE REVIEW

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/9e4ef902f18ce847b1ebb84859af825addb680d3/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/681d836c312c9bf6c7b930c0f42d8a0ddcb7efb3/index.html

File: packages/card/src/__tests__/shared.js [hunk 1]
```diff
@@ -14,178 +15,204 @@ const cardProps = {
...
+
+    component.setProps({
+      image: { uri: testUri }
+    });
+
+    expect(
+      component
+        .find("TimesImage")
+        .at(0)
+        .props().uri
```
Comment (jamesseanwright) [inline packages/card/src/__tests__/shared.js]: Is there any reason for calling `props()` over `prop('uri')`?

Comment (colincclark) [reply packages/card/src/__tests__/shared.js]: In a word, no. Have changed it

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/97c45124a7a7d7daa04ed0d754aace0fab714a63/index.html

**Comment by times-tools** [comment]
Please find visual snapshots of your changed components here: https://s3-eu-west-1.amazonaws.com/times-components-snaps/57df4a405193e41027ddda8160cce7cff1cb7ff8/index.html

**Pull request merged**